recipes-core/images/iot2050-image-swu-example.bb

```diff
@@ -40,6 +40,6 @@ IMAGE_INSTALL += "efibootguard libebgenv0"
 IMAGE_INSTALL += "customizations-swupdate"
 IMAGE_INSTALL += "swupdate-handler-roundrobin"
 IMAGE_INSTALL += "swupdate-complete-update-helper"
-IMAGE_INSTALL += "iot2050-watchdog"
+IMAGE_INSTALL += "${@ '' if d.getVar('QEMU_IMAGE') == '1' else 'iot2050-watchdog' }"
```

Comment:
Will things break if that is left in? But, yeah, we are missing some watchdog, see also https://gitlab.com/cip-project/cip-core/isar-cip-core/-/issues/66

Reply:
I think it will not be a problem. However, I thought that removing the package for qemu would make no watchdog support feature more visible. What do you suggest at the end?

Reply:
Change as little as needed to get a system starting that shows no errors.

Reply:
recipes-app/board-conf-tools/board-conf-tools_0.1.bb
This package could be removed as it gives error at the end of start process. However, it does not ruin booting process.

Reply:
Right, but that would be a good example for removing something that cannot work (given the existing device model in QEMU). Here, I would vote for removing, also to avoid confusion.

Reply:
Done.